docs/datatypes/stream/zstream.md

```diff
@@ -1475,7 +1475,7 @@ In the example below, we are partitioning incoming elements into three streams u
 val partitioned: ZIO[Scope, Nothing, (UStream[Int], UStream[Int], UStream[Int])] =
   ZStream
     .iterate(1)(_ + 1)
-    .fixed(1.seconds)
+    .scheduleFixed(1.seconds)
```

Comment:
Should this be `scheduleElementsFixed` (rather verbose but if it is `scheduleElements(Fixed...)`...

Reply:
Probably though it is a little bit long.